### PR TITLE
Adding a warning before starting on update operation

### PR DESCRIPTION
**Pull request by bganapa**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/StackAdmin/AzureStack/AzureStack.psd1 [hunk 1]
```diff
@@ -65,7 +65,7 @@
         @{ModuleName = 'Azs.Storage.Admin'; RequiredVersion = '0.2.0'; },
         @{ModuleName = 'Azs.Subscriptions'; RequiredVersion = '0.2.0'; },
         @{ModuleName = 'Azs.Subscriptions.Admin'; RequiredVersion = '0.3.0'; },
-        @{ModuleName = 'Azs.Update.Admin'; RequiredVersion = '0.2.0'; })
+        @{ModuleName = 'Azs.Update.Admin'; RequiredVersion = '0.3.0'; })
```
Comment (deathly809) [inline src/StackAdmin/AzureStack/AzureStack.psd1]: 0.2.1, VERY minor update.

File: src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1 [hunk 2]
```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.Update.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.2.0'
```
Comment (deathly809) [inline src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1]: 0.2.1, VERY minor update.

File: src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1 [hunk 3]
```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
...
+    ModuleVersion     = '0.2.1'
```
Comment (cormacpayne) [inline src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1]: @bganapa @deathly809 do we want to update the `ReleaseNotes` field in this file with the same information that was put into the change log for this version?

Comment (bganapa) [reply src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1]: Thanks. Done

**Pull request merged**